compiler/cli/cli-common/src/org/jetbrains/kotlin/cli/common/arguments/K2JSCompilerArguments.kt

```diff
@@ -25,11 +25,11 @@ class K2JSCompilerArguments : CommonCompilerArguments() {
     }
 
     @GradleOption(DefaultValues.StringNullDefault::class)
-    @Argument(value = "-output", valueDescription = "<path>", description = "Output file path")
+    @Argument(value = "-output", valueDescription = "<filepath>", description = "Destination *.js file for the compilation result")
     var outputFile: String? by NullableStringFreezableVar(null)
 
     @GradleOption(DefaultValues.BooleanTrueDefault::class)
-    @Argument(value = "-no-stdlib", description = "Don't use bundled Kotlin stdlib")
+    @Argument(value = "-no-stdlib", description = "Exclude kotlin-stdlib-js.jar from the compilation dependencies")
```

Comment:
I think it would be better to avoid using file names here.

Reply:
I think it isn't correct to say "exclude" here. It's the way to say the compiler "Don't include default/bundled stdlib".

Reply:
We use file names in the JVM's desription of `-no-stdlib`, and IMHO it adds clarity there. Maybe just `kotlin-stdlib-js` here for JS would be enough

Reply:
Other questions, unrelated to this PR:
1. Do we really need this option in "public API"?
2. Does anybody except us and (our) build tools use it?
3. Could/should we add stdlib to deps by default in build tools too and have a way to provide another version.

In JS IR BE we are going to stop supporting this option.

Reply:
Yes, we do need some option like this to be able to control the compilation classpath precisely. This is needed for external build tools and plugins, not all of which are under our control. E.g. Bazel, Buck, Kobalt, etc.

There can't be a default way to add stdlib to dependencies in build tools, because at least some build tools do not use our compiler distribution where all libraries are alongside the compiler. So there's no standard way to find `kotlin-stdlib.jar` from the compiler.

BTW this option could've been instead something like `-with-stdlib=<path>`. But I don't see any reason to change it, the current way already works for all use cases ¯\\_(ツ)_/¯